packages/nuxi/src/utils/prepare.ts

```diff
@@ -68,6 +69,10 @@ export const writeTypes = async (nuxt: Nuxt) => {
     }
   }
 
+  for (const alias in nuxt.options.alias) {
+    tsConfig.include.push(aliases[alias])
+  }
```

Comment:
can we deduplicate this somehow? For example, we shouldn't add any aliases that are already covered by an existing glob.